sbws/lib/v3bwfile.py

```diff
@@ -437,6 +442,22 @@ def desc_bw_bur_from_results(results):
                 return r.relay_burst_bandwidth
         return None
 
+    @staticmethod
+    def cons_bw_from_results(results):
+        """Obtain the last descriptor bandwidth burst from the results."""
+        for r in reversed(results):
+            if r.relay_bandwidth is not None:
+                return r.relay_bandwidth
+        return None
+
+    @staticmethod
+    def cons_is_unmeasured_from_results(results):
```

Comment:
agree, though then would use a different pattern than the "desc" methods. Maybe we can leave it for #28684?

Reply:
I don't mind if you use abbreviations, but if you do, always use the same abbreviation. And document the abbreviations using comments.

One of the easiest ways to be consistent is to always use the full name. Then there is no need for any documentation.

Reply:
I don't think we should leave this decision for #28684, because his patch adds keys to the bandwidth file. So we will be stuck with those keys until the next major revision.